stock_dashboard_api/models/stocks_models.py

```diff
@@ -0,0 +1,48 @@
+from stock_dashboard_api.utils import pool as db
+
+
+class Stocks:
+    _table = 'public.stocks'
+
+    def __init__(self, name, company_name, pk=None):
+        self.pk = pk
+        self.name = name
+        self.company_name = company_name
+
+    @classmethod
+    def create(cls, name, company_name):
+        with db.Connection() as conn:
+            query = f"""INSERT INTO {cls._table} (name, company_name)
+                        VALUES (%(name)s, %(company_name)s)
+                        RETURNING id, name, company_name;"""
+            conn.cursor.execute(query, {'name': name, 'company_name': company_name})
+            pk, name, company_name = conn.cursor.fetchone()
+            return Stocks(pk=pk, name=name, company_name=company_name)
+
+    def update(self, name=None, company_name=None):
+        query = f"UPDATE {self._table} SET "
```

Comment:
Multiple concatenations is a pretty bad approach for memory, coz every time you do this - it creates new string, so in case `name` and `company_name` is not `None` this code will create about 9 string objects.
Use list with join